src/doc/src/reference/workspaces.md

```diff
@@ -112,6 +112,90 @@ external tools may wish to use them in a consistent fashion, such as referring
 to the data in `workspace.metadata` if data is missing from `package.metadata`,
 if that makes sense for the tool in question.
 
+### The `workspace.package` table
+
+The `workspace.package` table is where you define keys that can be
+inherited by members of a workspace. These keys can be inherited by
+defining them in the member package with `{key}.workspace = true`.
+
+Keys that are supported:
+
+|                |                 |
+|----------------|-----------------|
+| `authors`      | `categories`    |
+| `description`  | `documentation` |
+| `edition`      | `exclude`       |
+| `homepage`     | `include`       |
+| `keywords`     | `license`       |
+| `license-file` | `publish`       |
+| `readme`       | `repository`    |
+| `rust-version` | `version`       |
+
+- `license-file` and `readme` are relative to the workspace root
+- `include` and `exclude` are relative to your package root
+
+Example:
+```toml
+# [PROJECT_DIR]/Cargo.toml
+[workspace]
+members = ["bar"]
+
+[workspace.package]
+version = "1.2.3"
+authors = ["Nice Folks"]
+description = "..."
```

Comment:
I prefer to tend towards keeping examples closer to reality without too many elisions. This can help avoid confusion with new users who may misinterpret it. I think in this case, it wouldn't hurt to include a few words instead of an ellipsis. Perhaps something like:

```suggestion
description = "A short description of my package"
```